app/backend/src/clj/teet/notification/ion.clj

```diff
@@ -0,0 +1,30 @@
+(ns teet.notification.ion
+  (:require [teet.cooperation.cooperation-db :as cooperation-db]
+            [teet.cooperation.cooperation-notifications :as cooperation-notifications]
+            [teet.activity.activity-db :as activity-db]
+            [teet.environment :as environment]
+            [datomic.client.api :as d]
+            [teet.notification.notification-db :as notification-db]))
+
+(defn notify
+  "Given number of days before Application expiration
+  new notification is sent to active Activity PM and
+  consultant who entered the application into system
+  providing the Activity is not finished"
+  ([days]
+   (let [conn (environment/datomic-connection)
+         db (d/db conn)
+         applications-to-expire (cooperation-db/applications-to-be-expired db days)]
+     (map #(let [application-id (first %)
```

Comment:
nothing is transacted here?

How are the notifications saved.

Also how to determine if a notification is already sent fo an application and avoid sending it again.